package.json

```diff
@@ -20,9 +20,9 @@
     "angular-route": "1.5.11",
     "angular-sanitize": "1.5.11",
     "angular-toarrayfilter": "1.0.2",
-    "angular-ui-bootstrap": "1.3.3",
     "angular-ui-router": "0.4.3",
-    "bootstrap": "3.3.7",
+    "angular1-ui-bootstrap4": "^2.4.22",
+    "bootstrap": "4.0.0-beta",
```

Comment:
Beta 2 is out, let's use that :-)

```
$ npm view bootstrap dist-tags
{ latest: '3.3.7', next: '4.0.0-beta.2' }
```

https://blog.getbootstrap.com/2017/10/19/bootstrap-4-beta-2/